createlinks

```diff
@@ -46,3 +46,14 @@ $event = "trusted-networks-modify";
 
 templates2events("/etc/httpd/conf.d/roundcubemail.conf", $event);
 event_services($event, 'httpd' => 'reload');
+
+
+#--------------------------------------------------
+# actions for nethserver-sssd-save event
+#--------------------------------------------------
+
+my $event = "nethserver-sssd-save";
+
+templates2events("/etc/roundcubemail/config.inc.php",  $event);
+event_services($event, 'httpd' => 'reload');
```

Comment:
Are you sure we need to reload httpd?

I think if all settings impact on the PHP application Apache reload is not needed ...unless we change its configuration too: but I can't figure out how it happens if just a PHP script changes.